assets/css/app.scss

```diff
@@ -1,6 +1,19 @@
+@import "colors";
+@import "inter_ui";
 @import "route_ladder";
 @import "route_ladders";
 
+body {
+  background-color: $pale-grey;
+  font-family: "Inter UI", "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-size: 16px;
+}
+
+svg {
+  stroke: $dark;
+  fill: $dark;
```

Comment:
Having this selector this generic might cause problems, because everywhere we use every svg everywhere, including the route ladder roads+stops, has to override it. I was thinking of replacing `.m-route-ladder__close-icon` with `.m-icon__close` or something like that. Not a strong preference.

Reply:
Yeah, true. In that case I might say let's wait till we have a second close button to refactor it out.